kpatch-build/kpatch-build

```diff
@@ -165,7 +165,7 @@ find_parent_obj() {
 }
 
 find_kobj() {
-	arg=$1
+	arg=$(echo $1 |sed 's,^\S*\.\./,,g')
```

Comment:
No need to open a new pull request.  You can just rebase this patch and git push --force to the same branch.